framework/BaseClasses/BaseInterface.py

```diff
@@ -70,7 +70,7 @@ def handleInput(self, paramInput, variableGroups=None, globalAttributes=None):
       self.name = paramInput.parameterValues['name']
     else:
       self.raiseAnError(IOError, 'not found name for a '+self.__class__.__name__)
-    self.type = paramInput.getName()
+    #self.type = paramInput.getName() -> we set this in __init__, setting it here to the spec type seems sketchy
```

Comment:
I'm hesitant to remove this, since it was part of the BaseClasses for a long time. Obviously the tests are passing, so perhaps there's not an issue any more; I can remove it if you want.